templates/tc476356/application-template-dockerbuild.json

```diff
@@ -0,0 +1,382 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "ruby-helloworld-sample",
+    "creationTimestamp": null,
+    "annotations": {
+      "description": "This example shows how to create a simple ruby application in openshift origin v3"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
+      "apiVersion": "v1beta3",
```

Comment:
Is v1beta3 required for this testing? do we need to update it to use v1 instead?